README.md

```diff
@@ -87,7 +113,7 @@ MiniApp.shared().list { (result) in
 or
 
 ```swift
-MiniApp.shared(with: Config.getCurrent()).list { (result) in
+MiniApp.shared(with: Config.getCustom()).list { (result) in
```

Comment:
getCustom() method is not available? Maybe this was edited by mistake ?

Reply:
good catch

Reply:
In fact it is a custom object I define upper in the documentation. `Config` is not part of the SDK

Reply:
I agree, Config is not part of SDK, but we don't have getCustom() method right?

Reply:
read the full file it will be clearer to you